cmake/CMakeLists.txt

```diff
@@ -38,14 +41,13 @@ INSTALL(FILES
         OpenCogMacros.cmake
         COPYING-CMAKE-SCRIPTS
         DESTINATION
-        ${DATADIR}/cmake/)
+        ${CMAKE_INSTALL_DIR}/cmake)
```

Comment:
/cmake not needed here... the path is suppose to be /..../cmake/CogUtil

Reply:
Yes, but all opencog projects append cmake to the path they load from CogUtilConfig.cmake. It doesn't seem to cause any problem, so I decided it is not worth to update all the projects.